ui/job-view/headerbars/ActiveFilters.jsx

```diff
@@ -81,7 +81,7 @@ export default class ActiveFilters extends React.Component {
             <b>Active Filters</b>
           </span>
           {filterBarFilters.map(filter => (
-            <span className="filtersbar-filter" key={filter.key}>
+            <span className="filtersbar-filter" key={`${filter.key}${filter.value}`}>
```

Comment:
This key wasn't always unique, so had to add the ``value`` to do so for when you have multiple of the same filter.